rootfs/scheduler/__init__.py

```diff
@@ -954,10 +954,8 @@ def _delete_rc(self, namespace, name):
 
         return response
 
-    def _healthcheck(self, controller, path='/', port=8080, delay=30, timeout=1):
-        # FIXME this logic ideally should live higher up
-        app_type = controller['spec']['selector']['type']
-        if app_type not in ['web', 'cmd']:
+    def _healthcheck(self, controller, routable = False, path = '/', port = 5000, delay = 30, timeout = 5):  # noqa
```

Comment:
Yeah but for some reason my local flake didn't like it. Will have a look again why that happened